### DISCUSSION TITLE
feature req: allow config.add.path to Request Transformer plugin

### DISCUSSION THREAD

**Discussion by geertbrit** [Kong/kong#7210] [Ideas and feature requests]
Similar to `config.add.querystring` , etc. I'd like the ability to specify a prefix-path using the Request Transformer plugin. Something like `config.add.prefixPath=/api/4.0`

Thoughts?

**Comment by ahmadnassri** [comment]
@gebrits I removed my comment, after I re-read yours ... and now i'm confused that you agreed with me :smile: 

so to clarify:
- are you asking to apply the transformation on a specific path? if so, then #505 :+1: 
- are you asking to add the ability to extend the upstream path with a prefix? it not, then lets re-word this issue to reflect that more accurately.

thanks!

**Comment by geertbrit** [comment]
Ha! In all honesty I was asking to extend the upstream path with a prefix. Then I found out I could just hardcode the path in the `upstream_url`. 

I've removed my reply above since that was indeed confusing :smile:

**Comment by ahmadnassri** [comment]
@gebrits cool, so you're good with current setup :+1: 

#505 adds more power to plugins, which would actually make the original point valid (to allow to modify the path prefix based on certain conditions) so i'm gonna keep this open.
